kartothek/io_components/metapartition.py

```diff
@@ -1600,3 +1626,24 @@ def parse_input_to_metapartition(obj, metadata_version=None):
         raise ValueError("Unexpected type: {}".format(type(obj)))
 
     return mp
+
+
+def _ensure_valid_indices(secondary_indices, mp_indices):
+    if not secondary_indices:
+        if mp_indices:
+            raise ValueError(
```

Comment:
Big thanks for writhing nice, detailed exception messages 👍

Reply:
My pleasure